test/e2e/data/infrastructure-vsphere-supervisor/main/clusterclass/kustomization.yaml

```diff
@@ -12,3 +12,6 @@ patches:
   - target:
       kind: ClusterClass
     path: ./patch-namingstrategy.yaml
```

Comment:
Let's update flavorgen instead so these variables are not added to templates/cluster-template-topology-supervisor.yaml & templates/clusterclass-template-supervisor.yaml

Reply:
Make sense to me, will update in the next patch.